### ISSUE TITLE
YouTube - Add uploader_avatar_url field to the -J flag output

### ISSUE THREAD

**Issue by XiovV** [yt-dlp/yt-dlp#8559]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm requesting a site-specific feature
- [X] I've verified that I'm running yt-dlp version **2023.10.13** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Example URLs

https://youtu.be/SanYFUB_3-w

## Why?
The -J flag already outputs some information about a channel when fetching data about a video (channel name, follower count, id, url, for example.), so I think it would be convenient if we had a field which shows the avatar url.

Example:
```
yt-dlp -J https://youtu.be/SanYFUB_3-w
```
```json
"channel": "INEXED",
"channel_follower_count": 24600,
"uploader": "INEXED",
"uploader_id": "@INEXED",
"uploader_url": "https://www.youtube.com/@INEXED",
## Add this field
"uploader_avatar_url": "https://yt3.googleusercontent.com/GFqGqJci_JNKth-y4jtXqYP2tiGWJFrBKUsOAipFGWA8vAz4WJJlY8iMTJ84cCCHi5tSHGiq2A=s0"
"upload_date": "20230323",
"availability": "public",
"original_url": "https://youtu.be/SanYFUB_3-w",
```

## Workaround
I've managed to find a workaround by using `jq` and `select`:
```
yt-dlp --playlist-items 0 -J https://www.youtube.com/@INEXED | jq '.thumbnails[] | select(.id=="avatar_uncropped")'  
```
```
{
  "url": "https://yt3.googleusercontent.com/GFqGqJci_JNKth-y4jtXqYP2tiGWJFrBKUsOAipFGWA8vAz4WJJlY8iMTJ84cCCHi5tSHGiq2A=s0",
  "id": "avatar_uncropped",
  "preference": 1
}
```

However, there are two issues with this approach:
1. I need to use the channel's URL, instead of the video URL
2. It creates an additional step

## My personal use-case
Here's my use-case to help you understand the need for creating this feature request. 
I am working on a project where I need to fetch information about a video, such as: video title, video length, uploader name, uploader url and uploader avatar url. I tried using the -J flag, but it doesn't display the avatar url. However, I found a workaround which I showed above, and although it gets the job done, it's inconvenient to use as I first have to fetch the uploader's URL from the video, then run one more command to fetch the avatar url, which slows everything down and adds complexity. It would've been a lot more convenient if I could just extract the avatar url from the initial -J command. Thank you!

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
I would post the debug log but I don't think there's a point as I'm not reporting a bug or an issue.
```

**Comment by bashonly** [comment]
The video page only provides thumbnail URLs with a max resolution of 176x176, it does not provide the uncropped / full resolution image URL

**Comment by XiovV** [comment]
> The video page only provides thumbnail URLs with a max resolution of 176x176, it does not provide the uncropped / full resolution image URL

Yes I am aware of that, however I am not looking for video thumbnails, I'm looking for the avatar of the person who has uploaded that video.

**Comment by bashonly** [comment]
> I'm looking for the avatar of the person who has uploaded that video.

That's what I was talking about. From extracting a video (not channel page), only thumbnail versions of the avatar are available

**Comment by pukkandan** [comment]
> I've managed to find a workaround by using `jq` and `select`:

You can do this in a single command if that's sufficient for you

```
❯ yt-dlp --skip-download --exec "pre_process:yt-dlp %(uploader_url)q -I0 -O playlist:%%(thumbnails.-1.url)s" https://youtu.be/SanYFUB_3-w -q
https://yt3.googleusercontent.com/GFqGqJci_JNKth-y4jtXqYP2tiGWJFrBKUsOAipFGWA8vAz4WJJlY8iMTJ84cCCHi5tSHGiq2A=s0
```

**Comment by XiovV** [comment]
> > I've managed to find a workaround by using `jq` and `select`:
> 
> You can do this in a single command if that's sufficient for you
> 
> ```
> ❯ yt-dlp --skip-download --exec "pre_process:yt-dlp %(uploader_url)q -I0 -O playlist:%%(thumbnails.-1.url)s" https://youtu.be/SanYFUB_3-w -q
> https://yt3.googleusercontent.com/GFqGqJci_JNKth-y4jtXqYP2tiGWJFrBKUsOAipFGWA8vAz4WJJlY8iMTJ84cCCHi5tSHGiq2A=s0
> ```

This might work, thank you!

**Comment by Roadhog360** [comment]
Was about to open an issue and was looking for this feature too, for the purpose of displaying it in a bot's status.
